StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory+Card.swift

```diff
@@ -22,14 +22,22 @@ extension PaymentSheetFormFactory {
         )
         let shouldDisplaySaveCheckbox: Bool = saveMode == .userSelectable && !canSaveToLink
 
-        // Link can't collect phone.
-        let includePhone = !configuration.linkPaymentMethodsOnly
-            && configuration.billingDetailsCollectionConfiguration.phone == .always
-
-        let contactInformationSection = makeContactInformation(
-            includeName: false, // Name is included in the card details section.
-            includeEmail: configuration.billingDetailsCollectionConfiguration.email == .always,
-            includePhone: includePhone)
+        // Make Contact Information section
```

Comment:
To be honest, the code seemed cleaner before.

Reply:
Open to suggestions!  I didn't want to handle this case in the new helper method since afaict it's unique to cards (it wants a ContactSection but with custom Link logic and without the name field).